### PR TITLE
ASIM-5153 - Update PumpEquipmentDescription 

### PR DESCRIPTION
**Pull request by BeneBr**
Update SDK definition of PumpEquipmentDescription
    
Update the PumpEquipment in case descriptio. Was removed the user_defined_esp_table and esp_parameters. Furthermore, was added the esp_manufacturer and esp_model.
    
[ASIM-5153](https://eden.esss.co/jira/browse/ASIM-5153)

### CODE REVIEW

**Comment by nicoddemus** [comment]
Please use a proper title in the PR, GitHub's default often is not very good (you did kinda do this, the first phrase in the PR description looks like a title).

File: tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py [hunk 1]
```diff
@@ -506,9 +506,9 @@
     ),
     esp_number_of_stages=2,
     esp_reference_density=Scalar(1000.0, "kg/m3"),
-    user_defined_esp_table=TABLE_PUMP_DESCRIPTION,
```
Comment (nicoddemus) [inline tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py:509]: Why did you remove `user_defined_esp_table` and `esp_parameters`?

**Review by nicoddemus — CHANGES_REQUESTED** [review]
Please add a CHANGELOG entry.

File: tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py (hunk 1 above)
Comment (BeneBr) [reply tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py:509]: Sorry, this was a mistake.

**Review by prusse-martin — APPROVED** [review]

**Review by prusse-martin — APPROVED** [review]

File: tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py (hunk 1 above)
Comment (nicoddemus) [reply tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py:509]: As we realized later: the strange thing was that it was removed from this file, but not from the schema; that as been fixed since then.

**Review by nicoddemus — APPROVED** [review]

**Review by cauebs — APPROVED** [review]

**Review by prusse-martin — APPROVED** [review]
